### PR TITLE
security: Add policy

### PR DESCRIPTION
**Pull request by joncinque**
#### Problem

SPL programs that we deploy to mainnet deserve their own security policy.

#### Solution

Add the policy.  Note that token-22 is not included in this list just yet, and we can add that once it's been audited -- thoughts?

Most of this just links to the monorepo security policy, so we don't have to maintain two different sets of the same information.

### CODE REVIEW

**Review by mvines — APPROVED** [review]
great start, thanks

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+1. [Incident Response Process](#process)
+
+<a name="reporting"></a>
+## Reporting security problems to Solana
+
+**DO NOT CREATE AN ISSUE** to report a security problem. Instead, please send an
+email to security@solana.com and provide your github username so we can add you
+to a new draft security advisory for further discussion.
+
+Expect a response as fast as possible, within one business day at the latest.
```
Comment (t-nelson) [inline SECURITY.md:15]: we need to set reasonable expectations here (and in the monorepo!). 72hrs?

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+<a name="bounty"></a>
+## Security Bug Bounties
```
Comment (t-nelson) [inline SECURITY.md:18]: I think SPL will need it's own bounty list.  the monorepo ones don't quite apply 1:1

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+may still be valuable.
+
+Many programs, including
+[token-swap](https://github.com/solana-labs/solana-program-library/tree/master/token-swap/program)
+and [token-lending](https://github.com/solana-labs/solana-program-library/tree/master/token-lending/program),
+have been forked and deployed by prominent ecosystem projects, many of which
+have their own bug bounty programs.
+
+While we cannot guarantee a bounty from another entity, we can put you in touch
+with these teams, increasing your chances of receiving a bounty for your work.
```
Comment (t-nelson) [inline SECURITY.md:47]:
```suggestion
While we cannot guarantee a bounty from another entity, we can help determine who
may be affected and put you in touch the corresponding teams.
```

let's not get their hopes up :wink:

**Review by t-nelson — DISMISSED** [review]
great start!  we can tune this up as we work out the kinks in the monorepo policy

File: SECURITY.md (hunk 3 above)
Comment (joncinque) [reply SECURITY.md:47]: Much better, thanks

File: SECURITY.md (hunk 1 above)
Comment (joncinque) [reply SECURITY.md:15]: Sure! I can also put in a change for the monorepo for that at least

**Pull request merged**